content/spin/v2/redis-trigger.md

```diff
@@ -89,7 +89,14 @@ The JavaScript/TypeScript SDK doesn't currently support Redis components.  Pleas
 
 {{ startTab "Python"}}
 
-The Python SDK doesn't currently support Redis components.  Please [let us know](https://github.com/fermyon/spin-python-sdk/issues) if this is important to you.
+In Python, to handler needs to implement the [`inboundRedis`](https://fermyon.github.io/spin-python-sdk/wit/exports/index.html#spin_sdk.wit.exports.InboundRedis)
```

Comment:
```suggestion
In Python, to handler needs to implement the [`InboundRedis`](https://fermyon.github.io/spin-python-sdk/wit/exports/index.html#spin_sdk.wit.exports.InboundRedis) class, and override the `handle_message` method:
```

(I may have the intent or terminology wrong, feel free to fix however you think best, it just seemed like there was a word or phrase missing in the original) (plus the minor casing typo)